lib/engine-addon.js

```diff
@@ -8,6 +8,7 @@ var writeFile = require('broccoli-file-creator');
 var Babel = require('broccoli-babel-transpiler');
 var concat = require('broccoli-concat');
 var DependencyFunnel = require('broccoli-dependency-funnel');
+var defaults = require('lodash.defaults');
```

Comment:
But Ember-CLI 2.12 doesn't support Node 4, does it? I don't particularly care, I was just under the impression that our support policy was based on the Ember(-CLI) versions we supported. Regardless, now seems as good a time as any to drop 0.12.